lib/main.dart

```diff
@@ -1,18 +1,110 @@
 import 'package:flutter/material.dart';
 
 void main() {
-  runApp(const MyApp());
+  runApp(const CloneInstagramApp());
 }
 
-class MyApp extends StatelessWidget {
-  const MyApp({super.key});
+class CloneInstagramApp extends StatelessWidget {
+  const CloneInstagramApp({super.key});
 
   // This widget is the root of your application.
   @override
   Widget build(BuildContext context) {
-    return const MaterialApp(
+    return MaterialApp(
       title: 'Clone Instagram',
+      theme: ThemeData(
+        useMaterial3: true,
+      ),
+      home: const DashboardPage(),
     );
   }
 }
 
+class DashboardPage extends StatefulWidget {
+  const DashboardPage({super.key});
+
+  @override
+  State<DashboardPage> createState() => _DashboardPageState();
+}
+
+class _DashboardPageState extends State<DashboardPage> {
+  int _selectedIndex = 0;
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+        appBar: AppBar(
+          title: const Text('Clone Instagram'),
```

Comment:
All strings should be defined in a string source file. In the most simple way is define a class `SourceString` with `static const appName = "Clone Instagram"`
When you want to use it, just call `SourceString.appName`

Reply:
I changed